registry/remote/repository.go

```diff
@@ -776,8 +784,8 @@ func (s *blobStore) FetchReference(ctx context.Context, reference string) (desc
 	}
 }
 
-// generateDescriptor returns a descriptor generated from the response.
-func (s *blobStore) generateDescriptor(resp *http.Response, refDigest digest.Digest) (ocispec.Descriptor, error) {
+// generateBlobDescriptor returns a descriptor generated from the response.
+func generateBlobDescriptor(resp *http.Response, refDigest digest.Digest) (ocispec.Descriptor, error) {
```

Comment:
Yeah it's not about the change, it is just a general question. 😄

Reply:
Got it... though I still don't understand the question I'm afraid.  `generateBlobDescriptor` doesn't *ever* return a `Docker-Content-Digest`, since it's simply returning a `ocispec.Descriptor`.  I'm sure that's not exactly what you meant, but I'm unable to determine what you *did* intend to ask.  The only header I see that function look at is `Content-Type`.

Reply:
Sorry for being unclear. I actually mean, will the ECR server always return the `Docker-Content-Digest` header for blob `HEAD` and `GET` requests?

Reply:
Ah, now I get you -- yes ECR will always return that header.

cc: @michaelb990